packages/roosterjs-content-model-api/lib/modelApi/table/insertTableRow.ts

```diff
@@ -26,11 +26,17 @@ export function insertTableRow(
             table.rows.splice(insertAbove ? sel.firstRow : sel.lastRow + 1, 0, {
                 format: { ...sourceRow.format },
                 cells: sourceRow.cells.map(cell => {
+                    const cellFormat =
```

Comment:
This code is same with the previous one, try extract them into a public function